### ISSUE TITLE
The "Download" button on work show pages is too general 

### ISSUE THREAD

**Issue by hortongn** [uclibs/scholar_uc#521]
Note: This may require input from others if a drastic change to the download button is needed.

Screen readers read text around links to help understand what the link does. If there's no surrounding text and the link text is very general there's no way to work out what the link does.
Additional link text added with the TITLE attribute doesn't work in:

NVDA 2014.1 with IE11 (the link title isn't read when tabbing)
JAWS 14 with IE11 or Firefox 29 (the link title isn't read when tabbing)
WindowEyes 8.4 with IE11 (the link title isn't read when tabbing)

**Comment by jamesvanmil** [comment]
@hortongn Which heading/problem does this map to on the accessibility report? I'm looking for recommendations on what this *should* look like.

**Comment by hortongn** [comment]
I can't see the report right now, but I think the heading was something about ambiguous links.  The concern is that a screen reader will land on the download button and all the user will know is it's a link that downloads something.  There's no context for the user to know what they are downloading.

I don't think we necessarily want to change the buttons to read "Download this <title> file," but maybe there is some alt text or some other way to help the user know more about what they are downloading.

On Oct 23, 2015 12:03 PM, James Van Mil <notifications@github.com> wrote:

@hortongn<https://github.com/hortongn> Which heading/problem does this map to on the accessibility report? I'm looking for recommendations on what this should look like.

**Comment by jamesvanmil** [comment]
Alt text with something like "Download this #{curation_concern.title}" would be ideal, that's using HTML tag a, which doesn't support alt text.

Here's some discussion I found, and none of the examples are obvious solutions:
http://www.w3.org/TR/WCAG20-TECHS/H30.html

I think the best tack is to take some time to learn more about acessibility and especially screen readers. That I'm starting with that Lynda course I just emailed around.

**Comment by jamesvanmil** [comment]
If the timeline for these accessibility improvements allows, I suggest tabling this for issue #289

**Comment by lawhorkl** [comment]
Is this still an issue? Going through the source code, the download link uses the title attribute to insert the file title and/or filename into the link. Not sure if this is still an active issue.

Also looking at the source code for the show article view, there appears to be an error with the file information section. Doesn't affect the display but could be related to this issue,

Lines 253 - 256, <div class='modal fade modal-collection' title= Help id= rollbackHelp>

**Comment by lisa3711** [comment]
In cases where the title is present, the download button pulls the title so that the pages says... "Download <title name>. However in some cases, a work might not have a title, in which case, the Download button says "Download ' ' "
Should have it so that if user specifies no title (adding another work to file), should pull in the title. 
Linda example

**Comment by lisa3711** [comment]
@lawhorkl on above issue where you comment "  Lines 253 - 256, <div class='modal fade modal-collection' title= Help id= rollbackHelp>"
what file were you looking at?

**Comment by lisa3711** [comment]
I think we can close this. With Issue #409 completed, the Download always now has a file name, as #409 fixed that. Therefore, this is no longer relevant.
